auth/realm/token/src/main/java/org/wildfly/security/auth/realm/token/TokenSecurityRealm.java

```diff
@@ -183,13 +183,17 @@ public SupportLevel getCredentialAcquireSupport(Class<? extends Credential> cred
 
         @Override
         public SupportLevel getEvidenceVerifySupport(Class<? extends Evidence> evidenceType, String algorithmName) throws RealmUnavailableException {
-            if (exists() && isBearerTokenEvidence(evidenceType)) {
+            if (isBearerTokenEvidence(evidenceType)) {
                 return SupportLevel.SUPPORTED;
             }
 
             return SupportLevel.UNSUPPORTED;
         }
 
+        private void setClaims(Attributes claims) throws RealmUnavailableException {
+            this.claims = claims;
+        }
+
         private Attributes getClaims() throws RealmUnavailableException {
             if (this.claims == null) {
                 this.claims = validateToken(this.evidence);
```

Comment:
Since `validateToken` now sets the `claims` value, I think this line can be changed to just `validateToken(this.evidence)`.